CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Improve web-components build process
 - Use new designs for smart camera web cdn
+- Deprecate version 1
```

Comment:
Can we be more specific on what's deprecated?

Reply:
Done